### PR TITLE
Do not have repeated classifications

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR presents the display of repeated classifications.

### CODE REVIEW

File: static/js/components/ClassificationList.jsx [hunk 1]
```diff
@@ -116,6 +117,10 @@ const ClassificationList = () => {
       } else {
         taxname = "Unknown taxonomy";
       }
+      if (usedClassifications.includes(classification)) {
```
Comment (profjsb) [inline static/js/components/ClassificationList.jsx:120]: It's ok to have the same classification if it's in a different (latest) taxonomy.

**Review by profjsb — CHANGES_REQUESTED** [review]

File: static/js/components/ClassificationList.jsx (hunk 1 above)
Comment (mcoughlin) [reply static/js/components/ClassificationList.jsx:120]: Yeah but we wouldn't want to redisplay... right?

File: static/js/components/ShowClassification.jsx [hunk 2]
```diff
@@ -37,57 +95,47 @@ export const getSortedClasses = (classifications) => {
...
-                className={classes.chip}
-              />
-            </Tooltip>
-          ));
-        })}
-      </div>
-    );
-  }
-  return <span />;
+  console.log("classificationsGrouped", classificationsGrouped);
```
Comment (guynir42) [inline static/js/components/ShowClassification.jsx:126]:
```suggestion
```

**Review by guynir42 — APPROVED** [review]
There's one console that shouldn't be there but otherwise looks ok.

**Pull request merged**